db/migrate/20250330143647_add_counter_caches_to_events.rb

```diff
@@ -7,7 +7,25 @@ def change
     reversible do |dir|
       dir.up do
         say_with_time "Updating Event counter caches..." do
-          Event.find_each { |event| Event.reset_counters(event.id, :sessions, :rooms, :timeslots) }
+          # Use SQL directly to avoid model scopes
+          execute <<-SQL
+            UPDATE events
+            SET sessions_count = (
+              SELECT COUNT(*)
+              FROM sessions
+              WHERE sessions.event_id = events.id
+            ),
```

Comment:
This migration was used as a one-time migration to populate counter caches, but the combination of a `default_scope` with `Event.reset_counters(event.id, :sessions, :rooms, :timeslots) }` [caused problems](https://github.com/minnestar/sessionizer/actions/runs/14683890245/job/41210021092) because `sessions.canceled_at` didn't exist at this point in the migrations.

```
Caused by:
PG::UndefinedColumn: ERROR:  column sessions.canceled_at does not exist (PG::UndefinedColumn)
LINE 1: SELECT "sessions".* FROM "sessions" WHERE "sessions"."cancel...
```